src/VisualStudio/Core/Def/Implementation/NavigationBar/NavigationBarClient.cs

```diff
@@ -152,7 +152,7 @@ int IVsDropdownBarClient.GetComboTipText(int iCombo, out string pbstrText)
             int selectionIndex;
             var selectedItemPreviewText = string.Empty;
 
-            if (_dropdownBar.GetCurrentSelection(iCombo, out selectionIndex) == VSConstants.S_OK)
+            if (_dropdownBar.GetCurrentSelection(iCombo, out selectionIndex) == VSConstants.S_OK && selectionIndex >= 0)
```

Comment:
Partially related.  While debugging the original issue, there was a case where the mis-matched content type could cause GetCurrentSelection() to return true but set selectionIndex to -1 which would then crash VS for a different reason due to the invalid index.  I figured since it was such a small fix I'd rather take care of it here than create another PR for it.